tests/webview/ui/test_content.py

```diff
@@ -769,14 +768,47 @@ def test_chapter_review_version_matches_book_version(webview_base_url, selenium,
     assert content.page_version == content.book_version
 
 
+@markers.webview
+@markers.test_case('C195064')
+@markers.nondestructive
+@markers.parametrize('ch_review_id', ['4fGVMb7P@1'])
+def test_books_containing_go_to_book_link(webview_base_url, selenium, ch_review_id):
+    # GIVEN the webview base url, a chapter review id, and the Selenium driver
+    content = ContentPage(selenium, webview_base_url, id=ch_review_id).open()
+    books = content.books_containing.book_list
+
+    # WHEN we click the link to the first book
+    title = books[0].title
+
+    book = books[0].click_go_to_book_link
+
+    # THEN the chapter should be the very first module 1.1
+    assert type(book) == Content
+    assert book.chapter_section == '1.1'
+    assert book.title == title
+
+
+@markers.webview
+@markers.test_case('C195063')
+@markers.nondestructive
+@markers.parametrize('ch_review_id', ['SjdU64Og@4'])
+def test_contain_revised_date(webview_base_url, selenium, ch_review_id):
+    # GIVEN the webview base url, a chapter review id, and the Selenium driver
+    content = ContentPage(selenium, webview_base_url, id=ch_review_id).open()
```

Comment:
may want to move content and books list to under the `WHEN` and make the `WHEN` comment:

`#WHEN the content_page is fully loaded and we have a list of books containing the page`